app/views/about/hiring_community.html.haml

```diff
@@ -0,0 +1,153 @@
+= render ViewComponents::AboutNav.new(:hiring)
+
+#page-hiring-job.bg-backgroundColorA.pb-16
+  %header.c-shapes
+    .md-container
+      = link_to hiring_about_path, class: "c-prominent-link" do
+        = graphical_icon "arrow-left"
+        Back to All Job Listings
+
+      .role
+        = graphical_icon "hiring-community", category: 'graphics', css_class: 'graphic'
+        .information
+          %h1.text-h0.mb-12 Community Manager
+          .details
+            .remote-tag
+              = graphical_icon "stars"
+              100% remote
+            .salary Salary: £50k (~$68k, ~€60k)
+
+  %article
+    .sm-container
+      %p.text-p-2xlarge.mb-40
+        %strong Become Exercism's Community Manager!
+        We need someone to lead, grow and nurture our community of students, mentors and contributors.
+        If you're passionate about community, love people, and are excited about taking a systematic approach to creating a kind, supportive and helpful community, then this role might be perfect for you!
```

Comment:
You don't strictly have to love people :) Feel free to ignore this suggestion (it does make for a slightly long line):

```suggestion
        If you're passionate about community, love working with people from all over the world, and are excited about taking a systematic approach to creating a kind, supportive and helpful community, then this role might be perfect for you!
```